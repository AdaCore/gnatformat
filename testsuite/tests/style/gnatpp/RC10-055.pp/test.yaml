description: |  
  Imported GNATpp test
driver: gnatformat
args: [
       "lf.ads",
       "crlf.ads",
       "mixed.ads",
       "--pipe"
      ]
