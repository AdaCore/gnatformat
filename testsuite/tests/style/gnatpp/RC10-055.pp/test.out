--  lf.ads
--  This file has LF line endings.

package LF is

   procedure P;
   procedure Q;

end LF;

--  crlf.ads
--  This file has CRLF line endings.

package CRLF is

   procedure P;
   procedure Q;

end CRLF;

--  mixed.ads
--  This file has mixed line endings.

package Mixed is

   procedure P;
   procedure Q;

end Mixed;

